ansible/roles/ceph-grafana/tasks/grafana_plugins.yml

```diff
@@ -15,3 +15,6 @@
 - name: Update Grafana plugins
   command: "grafana-cli plugins update {{ item }}"
   with_items: "{{ grafana_plugins }}"
+
+- name: Update status-panel for readability within the 'light' theme
+  command: "sed -i.bak -e 's/green/rgb(1,167,1)/g' /var/lib/grafana/plugins/vonage-status-panel/dist/css/status_panel.css"
```

Comment:
Also:
```
TASK [ceph-grafana : Update status-panel for readability within the 'light' theme] ***************************************
 [WARNING]: Consider using template or lineinfile module rather than running sed
```